### PR TITLE
update text for Domain joined check

### PR DESCRIPTION
**Pull request by MainiSuruchi**
update text for Domain joined check

### CODE REVIEW

File: AzFilesHybrid/AzFilesHybrid.psm1 [hunk 1]
```diff
@@ -3755,10 +3755,16 @@ function Debug-AzStorageAccountAuth {
         
                 if (!(Get-IsDomainJoined))
                 {
-                    $message = "Machine is not domain-joined. Mounting to Azure Files through" `
-                        + " Active Directory Authentication is only supported when the computer is joined to" `
-                        + " an Active Directory domain, which is synced to Azure AD with Azure AD Connect" `
-                        + " (https://docs.microsoft.com/en-us/azure/storage/files/storage-files-identity-auth-active-directory-enable#prerequisites)"
+                    $message = "Machine is not domain-joined." `
+                        + "Machine is not domain-joined.\n\n" `
+
```
Comment (MaximeKjaer) [inline AzFilesHybrid/AzFilesHybrid.psm1:3760]: No need to skip a line here.

File: AzFilesHybrid/AzFilesHybrid.psm1 [hunk 2]
```diff
@@ -3755,10 +3755,16 @@ function Debug-AzStorageAccountAuth {
         
                 if (!(Get-IsDomainJoined))
                 {
-                    $message = "Machine is not domain-joined. Mounting to Azure Files through" `
-                        + " Active Directory Authentication is only supported when the computer is joined to" `
-                        + " an Active Directory domain, which is synced to Azure AD with Azure AD Connect" `
-                        + " (https://docs.microsoft.com/en-us/azure/storage/files/storage-files-identity-auth-active-directory-enable#prerequisites)"
+                    $message = "Machine is not domain-joined." `
```
Comment (MaximeKjaer) [inline AzFilesHybrid/AzFilesHybrid.psm1:3758]: Remove the repetition of "Machine is not domain-joined"

File: AzFilesHybrid/AzFilesHybrid.psm1 [hunk 3]
```diff
@@ -3755,10 +3755,16 @@ function Debug-AzStorageAccountAuth {
...
-                    $message = "Machine is not domain-joined. Mounting to Azure Files through" `
-                        + " Active Directory Authentication is only supported when the computer is joined to" `
-                        + " an Active Directory domain, which is synced to Azure AD with Azure AD Connect" `
-                        + " (https://docs.microsoft.com/en-us/azure/storage/files/storage-files-identity-auth-active-directory-enable#prerequisites)"
+                    $message = "Machine is not domain-joined." `
+                        + "Machine is not domain-joined.\n\n" `
+ 
+                        + "Being domain-joined to an AD DS domain is a prerequisite for mounting" `
+                        + " Azure file shares without having to explicitly provide user credentials at every mount." `
+                        + " (https://docs.microsoft.com/en-us/azure/storage/files/storage-files-identity-auth-active-directory-enable#prerequisites).\n\n" `
```
Comment (MaximeKjaer) [inline AzFilesHybrid/AzFilesHybrid.psm1:3763]: Let's edit the links to be consistent between here and the one below.

I would suggest not skipping a line before the link, but instead just appending the sentence "See [link]" to the end of the paragraph.

File: AzFilesHybrid/AzFilesHybrid.psm1 (hunk 1 above)
Comment (MainiSuruchi) [reply AzFilesHybrid/AzFilesHybrid.psm1:3760]: fixed

File: AzFilesHybrid/AzFilesHybrid.psm1 (hunk 2 above)
Comment (MainiSuruchi) [reply AzFilesHybrid/AzFilesHybrid.psm1:3758]: fixed

File: AzFilesHybrid/AzFilesHybrid.psm1 [hunk 4]
```diff
@@ -3755,10 +3755,11 @@ function Debug-AzStorageAccountAuth {
         
                 if (!(Get-IsDomainJoined))
                 {
-                    $message = "Machine is not domain-joined. Mounting to Azure Files through" `
-                        + " Active Directory Authentication is only supported when the computer is joined to" `
-                        + " an Active Directory domain, which is synced to Azure AD with Azure AD Connect" `
-                        + " (https://docs.microsoft.com/en-us/azure/storage/files/storage-files-identity-auth-active-directory-enable#prerequisites)"
+                    $message = "Machine is not domain-joined." `
+                        + " Being domain-joined to an AD DS domain is a prerequisite for mounting" `
+                        + " Azure file shares without having to explicitly provide user credentials at every mount.See(https://docs.microsoft.com/en-us/azure/storage/files/storage-files-identity-auth-active-directory-enable#prerequisites).\n\n" `
```
Comment (MaximeKjaer) [inline AzFilesHybrid/AzFilesHybrid.psm1:3760]: I don't know if PowerShell will display the link properly if we put parentheses. Let's maybe go with: "See [link]" without parentheses

Comment (MainiSuruchi) [reply AzFilesHybrid/AzFilesHybrid.psm1:3760]: fixed

**Pull request merged**